Procurement/ViewModel/Filters/ForumExport/ExplicitModBase.cs

```diff
@@ -14,12 +14,29 @@ public ExplicitModBase(string keyword)
 
         public bool Applicable(Item item)
         {
-            if (item.Explicitmods == null || !(item is Gear))
+            Gear gear = item as Gear;
+            if (gear == null)
                 return false;
 
-            foreach (var mod in item.Explicitmods)
-                if (mod.Contains(keyword))
-                    return true;
+            if (gear.Explicitmods != null)
+                foreach (var mod in gear.Explicitmods)
+                    if (mod.Contains(keyword))
+                        return true;
+
+            if (gear.FracturedMods != null)
+                foreach (var mod in gear.FracturedMods)
+                    if (mod.Contains(keyword))
+                        return true;
+
+            if (gear.CraftedMods != null)
+                foreach (var mod in gear.CraftedMods)
+                    if (mod.Contains(keyword))
+                        return true;
+
+            if (gear.EnchantMods != null)
```

Comment:
Technically, enchant mods are not explicit mods (and you can argue crafted and fractured mods aren't either).  However, I think the intent of this class (as it is being used) is to search the mods of an item.  As such, I could see also adding implicit mods to this class, if we rename the class.

Ideally, which mods are search should be a parameter to the value/property being searched for and can be customized by the user.  But that is a much larger change that is outside the scope of this PR.

Reply:
If there is interest, it will be easy to support customized searches on search box with my other PR, like name: type: mod: explicit: implicit: crafted: fractured: enchanted: flavour: property: requirement: description: prophecy:

Reply:
It seems to be added with only explicit mods in mind, so if we're going to add implicit mods, I think it's better to remove the file and use StatFilter instead